lib/ansible/modules/windows/win_msg.ps1

```diff
@@ -0,0 +1,65 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2016, Jon Hawkesworth (@jhawkesworth) <figs@unity.demon.co.uk>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+$stopwatch = [system.diagnostics.stopwatch]::startNew()
+
+$params = Parse-Args $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+
+$display_seconds = Get-AnsibleParam -obj $params -name display_seconds -default "10" -failifempty $False -resultobj $result
+$msg = Get-AnsibleParam -obj $params -name msg -default "Hello world!" -resultobj $result
+$to = Get-AnsibleParam -obj $params -name to -default "*" -failifempty $False -resultobj $result
+$wait = Get-AnsibleParam -obj $params -name wait -default $False -failifempty $False -resultobj $result
+
+$result = @{
+    changed = $false
+}
+
+
+$msg_args = @($to, "/TIME:$display_seconds")
+
+If ($wait) 
+{
```

Comment:
@jborean93 There's no consensus, and there's no standard way of doing things in Powershell (cfr. PEP8 in Python). For that reason I removed all the formatting changes in my refactoring of most Windows modules on request of @nitzmahone 

I'd prefer a standard way of writing modules in absence of a higher authoritative source for clarity and conformance. But then again, I have no prior experience with Powershell and will have it easier to adapt to the "new normal" than others. (I do have preferences that are closer to the way C and Python does things :-))